src/dmd/cond.d

```diff
@@ -653,6 +653,10 @@ extern (C++) final class VersionCondition : DVCondition
             case "CRuntime_Microsoft":
             case "CRuntime_Musl":
             case "CRuntime_UClibc":
+            case "CppRuntime_DigitalMars":
+            case "CppRuntime_LibCpp":
```

Comment:
This is never used as far as I can see.

Reply:
> OSX? Clang distributes libc++ as standard no?

It's used on macOS yes.

> Oh, yeah.. Does DMD have any builds that build against such a C++ toolset?

Yes, on macOS.

Reply:
But for the C standard library on macOS there is no version identifier. In the code `OSX` is used.

Reply:
Why isn't there some lines configuring DMD for OSX right there next to the lines I edited?
What should the proper patch for OSX look like? Can you add a patch for OSX?

Reply:
Added for OSX + FreeBSD, they seem to be the 2 platforms that use it.